ext/standard/basic_functions.stub.php

```diff
@@ -836,6 +970,72 @@ function realpath_cache_get(): array {}
 
 function realpath_cache_size(): int {}
 
+/* formatted_print.c */
+
+/**
+ * @param mixed $format
+ * @param mixed ...$args
+ */
+function sprintf($format, ...$args): string {}
```

Comment:
We might want to make `$format` a string argument in the future.

Reply:
Yeah, it puzzled me as well why it is not a string. I'll do it in a separate commit then.

Reply:
I tried to convert the format param types to string, but there will be lots of failing tests (e.g. scanf_variation10 et al.) where I am not exactly sure what to do. Do these tests still necessary to have?